warehouse/templates/manage/account/recovery_codes-provision.html

```diff
@@ -61,7 +61,11 @@ <h2>{% trans %}Save your recovery codes{% endtrans %}</h2>
               <button type="button" class="button button--small copy-tooltip copy-tooltip-w" data-tooltip-label="{% trans %}Copy to clipboard{% endtrans %}" data-clipboard-text="{{ 'PyPI recovery codes\n' + recovery_codes|join('\n') }}">
                 {% trans %}Copy{% endtrans %}
               </button>
-              <a href="data:text/plain;,{{ recovery_codes|join('%0A') }}" download="PyPI-Recovery-Codes.txt" class="button button--small" data-tooltip-label="{% trans %}Download as file{% endtrans %}">
+              {% set _recovery_codes_filename = "PyPI-Recovery-Codes.txt" %}
+              {% if request.registry.settings.get("warehouse.domain") == "test.pypi.org" %}
```

Comment:
Replace this logic with `if testPyPI` to keep from duplicating the logic defined in [the base template](https://github.com/pypa/warehouse/blob/f6d1155f38502562487c2a65127c82e6ab5f3807/warehouse/templates/base.html#L74-L76).

Reply:
Ah, got it now. Done!